packages/token-introspection/src/openapi/token-introspection.yaml

```diff
@@ -0,0 +1,175 @@
+openapi: 3.1.0
+info:
+  title: Rafiki Authorization Server - Resource Server Connection
+  version: '1.0'
+  license:
+    name: Apache 2.0
+    identifier: Apache-2.0
+  summary: Rafiki Authorization Server - Resource Server Connection
+  description: 'The Open Payments API is secured via [GNAP](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol). This specification describes the connection between the Rafiki Open Payments Authorization Server and the Rafiki Open Payments Resource Server, which is an opinionated version of the [Grant Negotiation and Authorization Protocol Resource Server Connections](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers).'
+  contact:
+    email: tech@interledger.org
+servers:
+  - url: 'https://openpayments.guide/auth'
+tags:
+  - name: introspection
+    description: Token introspection
+paths:
+  /:
+    parameters: []
+    post:
+      summary: Introspect Access Token
+      operationId: post-introspect
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - properties:
+                      active:
+                        enum:
+                          - false
```

Comment:
Is this ok to have as an `enum`?

Reply:
Do you have a specific concern about `enum`?
I think I got the boolean as enum from this:
https://stackoverflow.com/a/16826238/12196980

Also, I think the [validation](https://github.com/kogosoftwarellc/open-api) [libraries](https://github.com/openapi-library/OpenAPIValidators) we're using don't fully support our OpenAPI 3.1 (just 3.0, which doesn't support `const`).

Reply:
No specific concern, just wanted to double check this would validate properly if given the correct `false` boolean :+1: